department.rb

```diff
@@ -0,0 +1,39 @@
+class Department
+
+  attr_reader :name
+
+  def initialize( name )
+    @name = name
+    @employees = []
+  end
+
+  def addEmployee( employee )
+    @employees << employee
+  end
+
+  def parseReviews
+    @employees.each do |employee|
+      employee.parseReviewText
+    end
+  end
+
+  def totalSalaries
+    return @employees.inject{|sum,x| sum.salary + x.salary }
```

Comment:
`inject`!  I see that the dark side of the force is strong with this one.

I jest, of course.  Inject is great, and a lot of people use it.  I like map/reduce because of the larger context of computer science, but collect/inject are cool.